.evergreen.yml

```diff
@@ -445,7 +479,58 @@ tasks:
         vars:
           node_js_version: "14.15.1"
 
+  ###
+  # E2E TESTS
+  ###
+  - name: e2e_tests_darwin
```

Comment:
Maybe this got lost when we switched to tag-based dependencies for tasks, but the draft publish task should also depend on these, right?

Reply:
Uh good point. Absolutely... I'll immediately add that. The arrows are in the picture but not in the YML :D :D